### ISSUE TITLE
All symbols must be updated

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [S-101-Portrayal-subWG/Working-Documents#124]
Recommended for PC 1.2.0.

All symbols must be reviewed and updated to remove inline CSS styling (not allowed by SVG Tiny 1.2), update the symbol metadata, and ensure conformance with updated SVG schema (once published).

* [x] Update symbols
* [ ] Register updated symbols

### Reference
[PC issue 177](https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/177)
https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/103#issuecomment-1448536951
https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/92#issuecomment-1419718682

**Comment by DavidGrant-NIWC** [comment]
@HolgerBothien has provided a stylesheet to do symbol conversion and conversion of all the registered symbols:
https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/103#issuecomment-1466481974

will need to be careful not to overwrite symbols which may have been changed within the PC but which aren't yet registered:
[PC issue 16](https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/16)

**Comment by DavidGrant-NIWC** [comment]
> @HolgerBothien wrote in issue #103
I can do that if someone provides me with the symbols to convert. A zip archive with all the SVGs to be converted would be great. The conversion is currently based on what I have proposed in the SVG profile and is using the namespace "http://www.iho.int/SVGMetadata/5.1" for the meta data. I don't know what the outcome of the TSM meeting was and would wait until it is crystal clear what should be used. If something needs to be modified I can do that but I have to know.

We can wait for the TSM 9 decisions and actions to be posted. We may need to do this in two steps:
1. Remove all the CSS inline styling so that the symbols conform to S-100 5.0
2. Change the namespace after the updated SVG profile is approved (hopefully for S-100 6.0)

**Comment by HolgerBothien** [comment]
I agree with the process done in steps. but the namespace used in the current symbol files needs to be changed anyway.

Used: http://www.iho.int/SVGMetadata
Correct according to S-100 Ed 5.0: http://www.iho.int/SVGMetadata/5.0

I can modify the XSLT to use the 5.0 namespace for the IHO metadata in SVG files,

**Comment by DavidGrant-NIWC** [comment]
> @HolgerBothien commented
> I agree with the process done in steps. but the namespace used in the current symbol files needs to be changed anyway.
> 
> Used: http://www.iho.int/SVGMetadata Correct according to S-100 Ed 5.0: http://www.iho.int/SVGMetadata/5.0

Fixed by [PC Issue 202](https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/202).

The following PC issues are intended to implement the remaining changes, just need a target version for each change:
[PC 177: Remove CSS inline styling from symbols](https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/177)
[PC 203: Review and update metadata for all PC elements](https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/203)
[PC 204: Update symbol namespaces for S-100 6.0 SVG schema](https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/204)

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 11/5/23**
- Wait for endorsement of S100 Ed 5.1.0 schemas before looking into updating existing registered symbols.

- [x] **NIWC** to proceed with https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/177 - **Target version PC 1.2.0**
- [x] **Alvaro** to include this topic in PsWG report to S101PT10 - Request KHOA GitLab issue is created to assess a back end, global updating of registered symbols to the new schema and their replacement within the GI Registry.
This covers https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/203 & https://github.com/iho-ohi/S-101_Portrayal-Catalogue/issues/204. - **Target version for this would be PC 2.0.0.**

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 18/10/23**

NIWC is still waiting on the endorsement of the S100 5.1.0 Schemas by the S100WG (expected during S100WG8 - November 2023).

- [ ] **NIWC** to update all PC symbols to the new schema once approved. 
- [ ] **NIWC** to update this issue when the task has been performed so it can be closed.

Note that symbols in the GI Registry will have to be bulk updated by KHOA/IHO Secretariat (together with other changes impacting SVG schemas, etc) at a later time (TBC - understanding is that this task is currently tracked in KHOA's GitLab space).

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 09/04/24**

NIWC is still waiting on the endorsement of the S100 5.2.0 Schemas by the S100WG.

- [x] **NIWC** to update all PC symbols to the new schema once approved. 
- [ ] **NIWC** to update this issue when the task has been performed so it can be closed.

Note that symbols in the GI Registry will have to be bulk updated by KHOA/IHO Secretariat or third party at a later time (TBC - understanding is that this task is currently tracked in KHOA's GitLab space).

 - [x] **Alvaro** to talk to S101PT Chair and recommend including this issue in his HSSC16 report.

**Comment by alvarosanuy** [comment]
**PsWG meeting on 11/7/2024**

There were discussions around the value of some SVG metadata fields; including their expected entries, potential sync issues with GI Registry Metadata, etc.
In short, the group decided that there's no official guidance on how all this should be managed and that, before making any changes to the symbol, the topic should be escalated to the S100WG for their consideration and guidance (e.g use S-99 as the instrument to document all GI Registry related procedures, including what elements are required to be registered (e.g Engineering Drawing for symbols - Yes or No? If Yes; what template to use?); what metadata is required and where is to be recorded, etc, etc).

**This issue can be closed with no further changes to PC.**

- [ ] **Alvaro** - Prepare a submission to S100WG9 (4-8 Nov 2024).

**Comment by iho-ohi** [comment]
[Bulk upload_Point sysmbol_ENC_Portrayal_register.xls](https://github.com/user-attachments/files/17992473/Bulk.upload_Point.sysmbol_ENC_Portrayal_register.xls)
Please review and complete the template for the bulk upload of point symbols to the Portrayal Register. Ensure that all revised symbols are provided as well. Once finalized, kindly submit the completed template along with all updated symbols to the IHO Secretariat (Yong and Jeff).

**Comment by alvarosanuy** [comment]
**PsWG meeting on 03/12/24**

- [x] **Yong** to provide the template for S101 PC Point symbols submission for GI Registry update (Phase 1)
- [ ] **NIWC** to complete template and provide PC 2.0.0 files to the IHO Secretariat (Yong and Jeff) soon after endorsement by MS (post 13/12/24). Close this issue when done.
